core/src/main/java/com/linecorp/armeria/client/DnsResolverGroupBuilder.java

```diff
@@ -297,6 +302,14 @@ public DnsResolverGroupBuilder decodeIdn(boolean decodeIdn) {
         return this;
     }
 
+    /**
+     * Sets {@link MeterRegistry} to collect the DNS query metrics.
+     */
+    DnsResolverGroupBuilder meterRegistry(MeterRegistry meterRegistry) {
```

Comment:
How about making this public?

Reply:
No, because we should use the `meterRegistry` specified in `ClientFactoryBuilder.meterRegistry()`.